stock_picking_auto_create_lot_sequence/tests/test_stock_picking_auto_create_lot_sequence.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2024 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import Form, TransactionCase
+
+
+class TestStockPickingProductLotSequence(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.supplier_location_id = cls.env.ref("stock.stock_location_suppliers").id
+        cls.stock_location_id = cls.env.ref("stock.stock_location_stock").id
+        cls.picking_type_in_id = cls.env.ref("stock.picking_type_in").id
```

Comment:
Might as well do this in the setup.
```suggestion
        picking_type_in = cls.env.ref("stock.picking_type_in")
        picking_type_in.auto_create_lot = True
```